pkg/deployer/rte/rte.go

```diff
@@ -37,16 +37,14 @@ type Options struct {
 }
 
 func SetupNamespace(plat platform.Platform) (*corev1.Namespace, string, error) {
-	if plat == platform.Kubernetes {
+	if plat == platform.Kubernetes || plat == platform.OpenShift {
```

Comment:
I guess this line can be omitted, we don't have another platform, even if it's unknown, it can be only one of the two options.

Reply:
yee, I think we can.

Reply:
yep. Let's drop this `if` entirely.

Reply:
done